bftengine/tests/bcstatetransfer/RVT_test.cpp

```diff
@@ -0,0 +1,110 @@
+// Concord
+//
+// Copyright (c) 2021 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the
+// LICENSE file.
+
+#include <iostream>
+#include <string>
+#include <random>
+#include <vector>
+#include <cryptopp/integer.h>
+
+#include "gtest/gtest.h"
+#include "Logger.hpp"
+
+#define HASH_SIZE 256
+
+using namespace std;
+using namespace CryptoPP;
+
+static std::string randomString(size_t length) {
+  static auto& chrs = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+  std::mt19937 rg{std::random_device{}()};
+  std::uniform_int_distribution<std::string::size_type> pick(1, sizeof(chrs) - 2);  // notice 0
```

Comment:
nit: comment is unclear. Not everyone would understand it. you should write:
'string literals are null teminated, and index starts from 0"

Reply:
Thanks. Addressed.